include/triton/Target/LLVMIR/Passes.h

```diff
@@ -5,6 +5,10 @@
 
 namespace mlir {
 
+namespace triton {
+enum Target { NVVM, ROCDL, Default = NVVM };
+} // namespace triton
+
```

Comment:
I've moved this enum from backend code to this header because of an multiple definitions error.

I am not sure this is the best place for this enum, but I did not find better option.

@ptillet Do we need this enum defined in backends in the first place?

Reply:
Hmm, where is this enum used, actually?

Reply:
Take a look at this PR: ihttps://github.com/openai/triton/pull/3230/

If you search through Nvidia backend code you can see that this variable actually is unused.

AMD backend has some uses that left since both backends were fused.